src/rpc/rawtransaction.cpp

```diff
@@ -555,6 +557,7 @@ static UniValue decodescript(const JSONRPCRequest& request)
                             {
                                 {RPCResult::Type::STR, "address", "segwit address"},
                             }},
+                            {RPCResult::Type::STR, "desc", "Inferred descriptor for the output\n"},
```

Comment:
There's no output here, just a raw script.

Reply:
can you give a suggestion?

Reply:
"Inferred output descriptor for the script" ?

Reply:
right I shortly realized what he meant, already pushed

Reply:
fixed